api/src/services/monitoring.js

```diff
@@ -189,7 +201,8 @@ const json = () => {
       outboundPushBacklog,
       outgoingMessageStatus,
       feedbackCount,
-      conflictCount
+      conflictCount,
+      ReplicationLimitLogs
```

Comment:
Nitpick: use a lower case "r".

```suggestion
      replicationLimitLogs
```

Reply:
😱 That's right. Fixed